lib/redis-mock.js

```diff
@@ -431,6 +432,12 @@ RedisClient.prototype.flushall = RedisClient.prototype.FLUSHALL = function (call
   serverfunctions.flushall.call(this, MockInstance, callback);
 }
 
+RedisClient.prototype.client = RedisClient.prototype.CLIENT = function () {
+  if (arguments[0].toLowerCase() === 'SETNAME') {
```

Comment:
toLowerCase() === 'SETNAME'  ? :)

Could we add GETNAME, LIST for the sake of completeness while we are at it?

One or two tests would make for a super happy new year too.